### DISCUSSION TITLE
Assets with variable paths

### DISCUSSION THREAD

**Discussion by revolxof** [stoatchat/discussions#78] [Feature Suggestions]
# Variable assets

Asset paths should be easy to change using the theme JSONs with their own key:
```jsonc
{
    // ...
    "assets": { "typing":"/path/to/typing/asset.gif" }
}
```

## Why should this be a feature

Having more customisable features means users can make revolt their own in a range wider than is currently possible

**Comment by ThatTonybo** [comment]
This could aid in adding extra elements for themes, such as new icons.
